steelthread-0/walkthrough.md

```diff
@@ -15,123 +18,234 @@ The installation has already been performed on an OpenShift cluster as per the i
-- Create a second address in the EnMasse console named ```work-queue/worker-updates``` choose the multicast option and the standard plan
+- Create a second address in the EnMasse console named ```work-queue/worker-updates``` choose the ```queue``` option and the Pooled Queue.
 
-# Setup Enmasse connection in Fuse Ignite
+# Deploy the CRUD REST HTTP booster
 
-Back in your ```evals``` project, You will see a binding and a binding secret as part of the Enmaase provisioned service. There is an option to view this secret. Click on view and click on reveal. Take note of the following values
+This is the HTTP endpoint that will revieve the requests from the Fuse Ignite 
+integreation we create and add new fruits to the inventory.
 
-- messagingHost
-- username
-- password
 
-open the Fuse Ignite route and login in to the Fuse Ignite console using the evals@example.com user 
+Load up the launcher application. This is at ```http://launcher-launcher.<ROUTER_SUFFIX> ``` 
 
-- Select connections and create connection
-- Find the AMQP connection and select it. 
-- Fill in the Connection URI Field with the a URI that is in the following format  ```amqp://<messagingHost>:5672?amqp.saslMechanisms=PLAIN```
-- Fill in the username and password fields
-- Disable Check Certificates
-- Click Validate 
-- Click the blue ```next``` button at the top right
-- Add a connection name ```enmaasse```
-- Click Next / Save
+- Click on launch your project
+- Login when prompted using the ```evals@example.com``` user
+- The create application form will show. Choose ```eval``` as the name (this equates to your namespace)
+- When prompted to select target environment choose : ```Code Locally, Build and Deploy```
+- Click the blue down arrows
+- Under mission select ```Crud```
+- Under runtime select ```Spring Boot```
+- Click the down arrows
+- Under ``` Authorize Git Provider``` you will need to authorize launcher to have access to your github account.
+- Once authorised, under the repository input change the value to ```crud-app```
+- Wait for the down arrows to change color to blue, then click them
+- Click setup application.
+- This will create a repo called crud-app in your github account and deploy the CRUD booster to the eval namespace.
 
+You can try out this application by going to the eval namepsace and clicking on the route created for this new application.
 
-# Deploy the messaging booster
 
-clone the following booster git@github.com:ssorj/nodejs-messaging-work-queue.git
+# Deploy the Messaging booster
 
-```
-git clone git@github.com:ssorj/nodejs-messaging-work-queue.git
+This is a webapp that connects to the EnMasse address space. It allows the user to add new fruit to the inventory of the CRUD app via the EnMasse and CRUD app Integration in Fuse.
 
-```
+Load up the launcher application. This is at ```https://launcher-launcher.<ROUTER_SUFFIX> ``` 
```

Comment:
At this point we cant tell the user to click the 'Start new project' button in the launcher console as they'll already be there from the previous section. Flows a bit better.